### ISSUE TITLE
Add device info to session

### ISSUE THREAD

**Issue by rishabhpoddar** [supertokens/supertokens-core#413]
## 🚀 Feature

Add device info in each session in the session data.

## Implementation details

We can store the device info in the session data.

**Comment by Avinashanakal** [comment]
Hey Rishabh can you give more details about the enhancement, i want to explore and start contributing to it

**Comment by Avinashanakal** [comment]
@rishabhpoddar thanks for the information.
In SessionInformationHolder we need to add new entry as device info?

**Comment by rishabhpoddar** [comment]
Not quite. You need to add the list in the accessTokePayload. This is one of the objects inside session information holder.
